flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/ResolvedSchema.java

```diff
@@ -63,15 +63,25 @@ public final class ResolvedSchema {
     private final List<Column> columns;
     private final List<WatermarkSpec> watermarkSpecs;
     private final @Nullable UniqueConstraint primaryKey;
+    private final List<Index> indexes;
 
     public ResolvedSchema(
```

Comment:
The ser and deser should also be adjust due to this new change. For example, `CompiledPlanSerdeUtil`